jobs/config.json

```diff
@@ -5072,8 +5072,8 @@
       "--gcp-node-image=gci",
       "--gcp-zone=us-central1-f",
       "--ginkgo-parallel",
-      "--gke-additional-zones=us-central1-a,us-central1-b",
       "--gke-environment=test",
+      "--gke-node-locations=us-central1-f,us-central1-a,us-central1-b",
```

Comment:
You cannot change this right away. Because you kubetest change needs to be rolled out before this flag is available.

Also, you need to remember to merge both the following changes together:
- kubetest image update (with your change)
- job config changes from old flag to new flag in order to not break them